README.md

```diff
@@ -18,6 +18,8 @@ Download the official installer for your operating system:
 You can install this alongside your existing GitHub Desktop for Mac or GitHub
 Desktop for Windows application.
 
+See our [getting started documentation](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/getting-started-with-github-desktop) for more information on how to set up, authenticate, and configure GitHub Desktop.
```

Comment:
Could we move this under the "More resources" header please?

Reply:
Yes, but I can't see a commit changes button, like I saw for your previous edit requests @niik

Reply:
Right, that's because suggested edit's, as they're called, work best for same line suggestions. They're a great tool for me to make small suggestions but for larger changes or, like in this case, moving a line somewhere else, we still have to resort to manually making the change and committing. You can update this branch by clicking on the `Files changes` tab and then selecting edit file.

![image](https://user-images.githubusercontent.com/634063/145223675-3eca672a-df2b-409d-ae51-2c950124568e.png)

That will give you a web editor if you would prefer to not make the change locally.

Reply:
I can not see the More Resources header.

Reply:
It's almost at the end of README.md (https://github.com/desktop/desktop/blob/05de6f4276c3b756271cdf68d3ce9847629a792d/README.md#more-resources)

Line 87

![image](https://user-images.githubusercontent.com/634063/145224420-8a62f3b0-165a-4718-a16f-804faf0809de.png)

Reply:
Done